### PR TITLE
360 Additional Checks blurb

### PR DESCRIPTION
**Pull request by edugomez**
#672

### CODE REVIEW

File: cove_360/templates/cove_360/explore.html [hunk 1]
```diff
@@ -173,17 +173,27 @@ <h3 class="panel-title panel-title-explore">
...
+              {% blocktrans %}Additional checks failed due to an issue with the data. Fix some validation errors first.{% endblocktrans %}
             </div>
           </div>
         </div>
         {% endif %}
         {% if additional_checks %}
+        <p class="explanation">{% blocktrans %}Additional checks have been made on your data.{% endblocktrans %}</p>
+        <br>
+        <p>
+            {% blocktrans %}
```
Comment (robredpath) [inline cove_360/templates/cove_360/explore.html]: Has this copy been signed off by 360?

Comment (edugomez) [reply cove_360/templates/cove_360/explore.html]: It is a minor variation of this:
https://github.com/OpenDataServices/cove/issues/672#issuecomment-299483191
by @KDuerden. 

It reads:

>Additional checks have been made on your data.
Additional checks highlight areas where the source data may be incorrect or require further attention. They can also suggest ways to increase the usefulness of the data. In any case, these checks do not indicate validation errors and can be ignored if they are not relevant.

@KDuerden, is that OK?

Comment (edugomez) [reply cove_360/templates/cove_360/explore.html]: @KDuerden I will add _"some"_ to that line. Any other final tweak before we consider this good to go?

Comment (KDuerden) [reply cove_360/templates/cove_360/explore.html]: Great - good to go. Thank you.

**Comment by edugomez** [comment]
@robredpath can you approve if OK?

**Pull request merged**